open-models/serving/cloud_run_ollama_gemma2_rag_qa.ipynb

```diff
@@ -0,0 +1,1007 @@
+{
```

Comment:
<p>Not a dealbreaker here, but it's very possible that Gemma 2 could have correctly answered these questions without RAG at all. I know it's not an educational notebook on RAG, but I feel the questions here should be specific to what's uniquely available in the documentation page vs what Gemma 2 might already know? If so, the whole point of setting up RAG here is a bit weak.</p><p> </p><p>Two possible ways to deal with this:</p><ol><li>Ask a very specific question about the documentation that is likely only found in the documentation, not in Gemma 2's closed domain.</li><li>Ask a question without RAG, and then the same question with RAG, to show RAG working well with Gemma 2.</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/968/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>

Reply:
Done, great suggestion!